ui/helpers/constants/gas.js

```diff
@@ -0,0 +1,27 @@
+export const GAS_FORM_ERRORS = {
+  GAS_LIMIT_OUT_OF_BOUNDS: 'editGasLimitOutOfBounds',
+  MAX_PRIORITY_FEE_TOO_LOW: 'editGasMaxPriorityFeeLow',
+  MAX_FEE_TOO_LOW: 'editGasMaxFeeLow',
+  MAX_PRIORITY_FEE_ZERO: 'editGasMaxPriorityFeeZeroError',
+  MAX_PRIORITY_FEE_HIGH_WARNING: 'editGasMaxPriorityFeeHigh',
+  MAX_FEE_HIGH_WARNING: 'editGasMaxFeeHigh',
+};
+
+export function getGasFormErrorText(type, t) {
```

Comment:
nit: Do we want to have functions in files with constants? I would probably move the function to `shared/modules/gas.utils.js`

Reply:
We use this pattern elsewhere; on another PR, Brad mentioned that putting things in shared bloats bundle sizes.  These constants aren't actually used in the background, thus this is fine.